### DISCUSSION TITLE
Welcome to comflowy Discussions!

### DISCUSSION THREAD

**Discussion by 6174** [6174/comflowy#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tianxu1998** [comment]
作者文章写的很易懂，适合初入门的小白。
